src/_common/community/card-base/AppCommunityCardBase.vue

```diff
@@ -1,64 +1,45 @@
-<script lang="ts">
-import { setup } from 'vue-class-component';
-import { Options, Prop, Vue } from 'vue-property-decorator';
+<script lang="ts" setup>
```

Comment:
import AppButton, and then remind me to show you again how to convert the translations to not use <AppTranslate> anymore